reorder_python_imports.py

```diff
@@ -522,22 +522,81 @@ def _report_diff(contents: str, new_contents: str, filename: str) -> None:
     ),
     ('py37', ('generator_stop',)),
 )
+MYPY_EXTENSIONS_IMPORTS: Tuple[Tuple[str, Tuple[str, ...]], ...] = (
+    (
+        'py37', (
+            'NoReturn',  # >= 3.6.2
+        ),
+    ),
+    (
+        'py38', (
+            'TypedDict',  # >= 3.8
+        ),
+    ),
+)
+TYPING_EXTENSIONS_IMPORTS: Tuple[Tuple[str, Tuple[str, ...]], ...] = (
+    (
+        'py36', (
+            'Collection',  # >= 3.6
+            'ContextManager',  # >= 3.6
+        ),
+    ),
+    (
+        'py37', (
+            'ChainMap',  # >= 3.6.1
+            'Counter',  # >= 3.6.1
+            'Deque',  # >= 3.6.1
+            'AsyncContextManager',  # >= 3.6.2
+            'AsyncGenerator',  # >= 3.6.2
+        ),
+    ),
+    (
+        'py38', (
+            'OrderedDict',  # >= 3.7.2
+            'Final',  # >= 3.8
+            'Literal',  # >= 3.8
+            'Protocol',  # >= 3.8
+            'SupportsIndex',  # >= 3.8
+            'final',  # >= 3.8
+            'get_args',  # >= 3.8
+            'runtime_checkable',  # >= 3.8
+        ),
+    ),
+)
 BUILTINS = (  # from python-future
     'ascii', 'bytes', 'chr', 'dict', 'filter', 'hex', 'input', 'int', 'list',
     'map', 'max', 'min', 'next', 'object', 'oct', 'open', 'pow', 'range',
     'round', 'str', 'super', 'zip',
 )
 
 
-def _add_future_options(parser: argparse.ArgumentParser) -> None:
+def _add_version_options(parser: argparse.ArgumentParser) -> None:
+    pys = sorted({
+        py for py, _ in
+        FUTURE_IMPORTS + MYPY_EXTENSIONS_IMPORTS + TYPING_EXTENSIONS_IMPORTS
+    })
+    f_index = dict(FUTURE_IMPORTS)
+    m_index = dict(MYPY_EXTENSIONS_IMPORTS)
+    t_index = dict(TYPING_EXTENSIONS_IMPORTS)
+
     prev: List[str] = []
-    for py, removals in FUTURE_IMPORTS:
+    for py in pys:
         opt = f'--{py}-plus'
-        futures = ', '.join(removals)
-        implies = f'. implies: {", ".join(prev)}' if prev else ''
+        implies = f'implies: {", ".join(prev)}' if prev else ''
+
+        msgs = []
+        if py in f_index:
+            tpl = 'Removes obsolete future imports ({}).'
+            msgs.append(tpl.format(', '.join(f_index[py])))
+        if py in m_index:
+            tpl = 'Imports mypy_extensions attrs from typing ({}).'
+            msgs.append(tpl.format(', '.join(m_index[py])))
+        if py in t_index:
+            tpl = 'Imports typing_extensions attrs from typing ({}).'
+            msgs.append(tpl.format(', '.join(t_index[py])))
+
         parser.add_argument(
-            opt, action='store_true',
-            help=f'Remove obsolete future imports ({futures}){implies}',
+            opt, action='store_true', help=' '.join((*msgs, implies)),
```

Comment:
there's also six imports -- I would probably drop all this from the help and just have `Removes/updates obsolete imports, assuming this version or newer`